demisto_sdk/commands/format/update_playbook.py

```diff
@@ -22,7 +22,19 @@ def __init__(self, input: str = '', output: str = '', path: str = '', from_versi
     def add_description(self):
         """Add empty description to playbook and tasks."""
         if self.verbose:
-            click.echo('Adding empty descriptions to relevant tasks')
+            click.echo('Adding descriptions for the playbook and to relevant tasks')
+        if 'description' not in set(self.data.keys()):
+            click.secho('No description is specified for this playbook, would you like to add a description? [Y/n]',
+                        fg='red')
```

Comment:
I suggest using `bright-red` for better readability - test it out see which looks better to you.